src/scss/objects/_form-controls.scss

```diff
@@ -0,0 +1,240 @@
+             transform: translate(-50%, -50%) rotate(45deg) scale(0);
+         }
+
+         &:after {
+             transform: translate(-50%, -50%) rotate(-45deg) scale(0);
+         }
+     }
+
+     .o-formControl-icon {
+         background-position: center 0;
+         left: spacing();
+         min-width: spacing(x1.5);
+         position: absolute;
+     }
+
+     .o-formControl-indicator--tickbox {
+         &:before {
+             top: 0;
+             transform: translateX(-50%) rotate(45deg) scale(0);
+             border: 2px solid $formControl-background-color;
+             background-color: transparent;
+             width: 40%;
+             height: 80%;
+             border-top: 0;
+             border-left: 0;
+         }
+
+         &:after {
+             display: none;
+         }
+     }
+
+     .o-formControl-input:checked ~ .o-formControl-indicator--checkbox,
+     .o-formControl-input:checked ~ .o-formControl-indicator--tickbox,
+     .o-formControl-input:checked ~ .o-formControl-indicator--radio {
+
+         label:hover & {
+             background-color: $formControl-hover-color;
+             border-color: $formControl-hover-color;
+         }
+     }
+
+     .o-formControl-input:checked ~ .o-formControl-indicator--checkbox,
+     .o-formControl-input:checked ~ .o-formControl-indicator--tickbox {
+         background-color: $formControl-color;
+         border: 3px solid $formControl-color;
+
+         &:before,
+         &:after {
+             opacity: 1;
+         }
+     }
+
+     .o-formControl-input:checked ~ .o-formControl-indicator--tickbox {
+         &:before,
+         &:after {
+             transform: translateX(-50%) rotate(45deg) scale(1);
+         }
+     }
+
+     .o-formControl-input:checked ~ .o-formControl-indicator--checkbox {
+         &:before {
+             transform: translate(-50%, -50%) rotate(45deg) scale(1);
+         }
+
+         &:after {
+             transform: translate(-50%, -50%) rotate(-45deg) scale(1);
+         }
+     }
+
+
+     // Radio modifiers
+     .o-formControl-indicator--radio {
+         border-radius: 50%;
+
+         // add a margin to align the radio where the line height is 1.6 as opposed to 1.2
+         @include media('<wide') {
+             margin-top: 2px;
+         }
+     }
+
+     .o-formControl-input:checked ~ .o-formControl-indicator--radio {
+         background-color: $formControl-color;
+         border: 6px solid $formControl-borderColor;
+
+         &:before {
+             content: '';
+             background-color: $formControl-color;
+             border: 3px solid $grey--lightest;
+             border-radius: 50%;
+             width: $formControl-width - 2;
+             height: $formControl-width - 2;
+             position: absolute;
+             top: -5px;
+             left: -5px;
+
+             @include media('>=mid') {
+                 width: $formControl-width--wide - 2;
+                 height: $formControl-width--wide - 2;
```

Comment:
`$formControl-height--wide`

Reply:
Might be better to change this to be `$formControl-diameter--wide` as it's a square, so the same value will be used for width/height

Reply:
@ashleynolan `$formControl-diameter--wide` to me would be referencing a circle (radio button) so for me this would be incorrect as the width is also for a square (checkbox).

Reply:
`length`?